### PR TITLE
Update install.txt

### PR DESCRIPTION
**Pull request by Sarangan0219**

### CODE REVIEW

**Comment by Arshardh** [comment]
Fixes https://github.com/wso2/product-apim/issues/8903

File: INSTALL.txt [hunk 1]
```diff
@@ -56,18 +57,35 @@ Running WSO2 API Manager in standalone mode
...
    medium or lower.
-5. To compile and run the sample clients, an Ant version is required. Ant 1.7.0
+5. To compile and run the sample clients in <APIM_HOME>/samples, an Ant version is required. Ant 1.7.0
    version is recommended
-6. To build WSO2 AM from the Source distribution, it is necessary that you have
+6. To build WSO2 APIM from the Source distribution, it is necessary that you have
    JDK 1.8 version and Maven 3.0.4 or later
 
+For more details see
+   https://apim.docs.wso2.com/en/next/install-and-setup/install/installation-prerequisites/
```
Comment (rmsamitha) [inline INSTALL.txt:89]: Isn't it ideal to link to the specific doc-version path itself? i.e.
https://apim.docs.wso2.com/en/3.2.0/install-and-setup/install/installation-prerequisites/

File: INSTALL.txt [hunk 2]
```diff
@@ -56,18 +57,35 @@ Running WSO2 API Manager in standalone mode
...
+
+There are sensitive information such as passwords in the carbon configuration.
+You can secure them by using secure vault. Please go through following steps to
+secure them with default mode.
+
+    On Linux: ./ciphertool.sh -Dconfigure
+    On Windows: ./ciphertool.bat -Dconfigure
+
+For more details see
+    https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/logins-and-passwords/working-with-encrypted-passwords/
```
Comment (rmsamitha) [inline INSTALL.txt:72]: Isn't it ideal to link to the specific doc-version path itself? 
i.e. https://apim.docs.wso2.com/en/3.1.0/install-and-setup/setup/security/logins-and-passwords/working-with-encrypted-passwords/

Comment (rmsamitha) [reply INSTALL.txt:72]: @bhathiya

Comment (bhathiya) [reply INSTALL.txt:72]: yes, we should do so whenever we use links in docs.

Comment (Sarangan0219) [reply INSTALL.txt]: Fixed https://github.com/wso2/product-apim/pull/8971/commits/061ead8cf43422243492a81c383c1681db180714

File: INSTALL.txt (hunk 1 above)
Comment (Sarangan0219) [reply INSTALL.txt]: Fixed https://github.com/wso2/product-apim/pull/8971/commits/061ead8cf43422243492a81c383c1681db180714

File: INSTALL.txt (hunk 2 above)
Comment (rmsamitha) [reply INSTALL.txt]: @Sarangan0219 pls fix this inn release-notes.html too.

Comment (Sarangan0219) [reply INSTALL.txt]: Fixed https://github.com/wso2/product-apim/pull/8971/commits/0fdb4d83b18bce93b8bc175a80ba28c40261a25b

**Pull request merged**